app/consumers/concerns/report_parsing.rb

```diff
@@ -24,20 +24,30 @@ def enqueue_parse_report_job(reports)
       reports.each do |profile_id, report|
         job = ParseReportJob
               .perform_async(ActiveSupport::Gzip.compress(report), metadata)
-        logger.info("Message enqueued: #{request_id} as #{job}" \
-                    " for #{profile_id}")
-        notify_payload_tracker(:received,
-                               "File is valid. Job #{job} enqueued")
+        notify_report_success(profile_id, job)
       end
 
       validation_payload(request_id, valid: true)
     end
 
     private
 
+    def notify_report_success(profile_id, job)
+      msg = "Enqueued report parsing of #{profile_id}"
+      msg += " from request #{request_id} as job #{job}"
+      logger.info(msg)
+      logger.audit_success(msg)
```

Comment:
Are these two not redundant?

Reply:
We've introduced a custom audit logger. The second call goes to that. The first does a regular app logging.